cmake/GoogleCloudCppFeatures.cmake

```diff
@@ -219,6 +219,10 @@ endfunction ()
 # additional samples that are enabled if needed.
 function (google_cloud_cpp_enable_features)
     foreach (feature ${GOOGLE_CLOUD_CPP_ENABLE})
+        if ("${feature}" STREQUAL "docfx")
+            add_subdirectory(docfx)
+            continue()
```

Comment:
If we're going to use `elseif` for all the other cases, we might as well do the same here.  (Or vice versa.)

Reply:
Fixed